### PR TITLE
[Big Bang] Introduce copy model

### PR DESCRIPTION
**Pull request by babsey**

### CODE REVIEW

File: src/components/node/NodeMenu.vue [hunk 1]
```diff
@@ -324,7 +324,7 @@ export default Vue.extend({
      * Reset node color.
      */
     const resetColor = () => {
-      state.node.view.color = null;
+      state.node.view.color = 'none';
```
[the remark this answers is not in the document]
Comment (babsey) [reply src/components/node/NodeMenu.vue:327]: ok, you can keep it:
`state.node.view.color = null;`

But can you check the code line 52 in `core/node/nodeViews.ts`?

[the remark this answers is not in the document]
Comment (jens-br) [reply src/components/node/NodeMenu.vue:327]: I pushed some additions to your NEST Desktop fork.

**Review by jens-br — APPROVED** [review]
LGTM :+1:

**Comment by babsey** [comment]
Closes #313

**Pull request merged**